### DISCUSSION TITLE
Plans to support RelativeSource AncestorType?

### DISCUSSION THREAD

**Discussion by sam-wheat** [microsoft/microsoft-ui-xaml#8208] [Q&A]
I am writing a templated control and I really miss `RelativeSource AncestorType`. 
Are there plans to implement it?
I am aware that a workaround exists in Community Toolkit.

Thanks

**Comment by parko65** [comment]
Have you tried something like this:

`<Ellipse x:Name="PART_Hub"
                                 Width="10"
                                 Height="10"
                                 Fill="{Binding HubColor, RelativeSource={RelativeSource Mode=TemplatedParent}}"/>`

**Comment by sam-wheat** [comment]
@parko65 Useful - but not what I am asking for.
